src/mergeProps.js

```diff
@@ -0,0 +1,27 @@
+import { curry, mergeAll, props } from 'ramda';
+
+/**
+ * Functional equivalent of merging object properties with object spread operator.
+ *
+ * @func mergeProps
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.17.0|v1.17.0}
+ * @category Object
+ * @sig [k] -> {k: {a}} -> {a}
+ * @see {@link RA.mergePaths|mergePaths}
```

Comment:
@char0n  mergePaths????

Reply:
Yup, mergeProps should link to mergePaths and vice versa. Utils should link to related utils, not to their implementation details. Does it make sense ?